docs/guides/getting-started-java-spring.md

```diff
@@ -133,7 +133,7 @@ See our documentation on [adding the Spring Zeebe SDK to your project](/apis-too
 </repositories>
 ```
 
-2. Add the following dependency your `pom.xml` file, as a child of the `<dependencies>` element:
+2. Add the following dependency to your `pom.xml` file, as a child of the `<dependencies>` element:
 
 :::warning
```

Comment:
This warning is still here and should likely be softened with updates to point at the patch. Let's soften the warning so we can get this merged and follow up when the patch is out.

Reply:
Do you just mean warning > caution, or also change the language?

e.g. change to:

:::caution
Due to a bug in 8.6, use the Spring Zeebe SDK version `8.5.0`.
:::

Or to:

:::caution
Due to an issue in 8.6, use the Spring Zeebe SDK version `8.5.0`. This issue will be resolved in a future patch release.
:::

Reply:
Added a commit - your first option is what I went with.